homework/index.js

```diff
@@ -30,18 +30,96 @@
+
+    const tr1 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr1, { text: 'Repository:' });
+    const repositoryCell = createAndAppend('td', tr1);
+    createAndAppend('a', repositoryCell, { text: data[value].name, href: data[value].html_url });
+
+    const tr2 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr2, { text: 'Description:' });
+    createAndAppend('td', tr2, { text: data[value].description });
+
+    const tr3 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr3, { text: 'Forks:' });
+    createAndAppend('td', tr3, { text: data[value].forks_count });
+
+    const tr4 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr4, { text: 'Update:' });
+    createAndAppend('td', tr4, { text: data[value].updated_at });
+  }
+
+  function drawRepositoryContributors(data, value, container) {
+    const contributorURL = data[value].contributors_url;
+    const rightDiv = createAndAppend('div', container, { class: 'right-div' });
+    createAndAppend('p', rightDiv, { class: 'contributor-header', text: 'Contributions' });
+    const ul = createAndAppend('ul', rightDiv, { class: 'contributor-list' });
+
+    fetchJSON(contributorURL, (err, contributorData) => {
+      contributorData.forEach((entry, index) => {
+        const li = createAndAppend('li', ul, { class: 'contributor-item', tabindex: index });
+        createAndAppend('img', li, { src: entry.avatar_url, height: '48', class: 'contributor-avatar' });
+        const div = createAndAppend('div', li, { class: 'contributor-data' });
+        createAndAppend('div', div, { text: entry.login });
+        createAndAppend('div', div, { text: entry.contributions });
+      })
+    });
+  }
+  // function rightContainer(url, data) {
+  //   const rightDiv = document.getElementsByClassName('right-container');
+  //   rightDiv.innerHTML = '';
+  //   // const paragraph = createAndAppend('p', rightDiv, { text: 'contributions' }, { class: 'contibutors-header' });
+  //   // const unlist = createAndAppend(ul, paragraph, { class: 'contributors-list' });
+  // }
+
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        // sort the data
+        data.sort((a, b) => {
+          const x = a.name.toLowerCase();
+          const y = b.name.toLowerCase();
+          if (x < y) {
+            return -1;
+          }
+          if (x > y) {
+            return 1;
+          }
+          return 0;
+        });
+
+        const header = createAndAppend('header', root, { class: 'header' });
+        const selectElem = createAndAppend('select', header, { class: 'repo-selector' });
+        // for each entry in the data, add option elements
+        data.forEach((entry, index) => {
```

Comment:
Now here if you followed my last suggestion, you will find here you are just writing english, as the following:

```suggestion
        reposories.forEach((repository, index) => {
```